frontend/src/metabase-lib/v1/expressions/helper-text-strings.ts

```diff
@@ -618,71 +618,112 @@ const HELPER_TEXT_STRINGS: HelpTextConfig[] = [
   {
     name: "contains",
     structure: "contains",
-    description: () => t`Checks to see if string1 contains string2 within it.`,
+    description: () =>
+      t`Returns true if string1 contains string2 within it (or string3, etc. if specified).`,
```

Comment:
I'm surprised to learn that it uses `OR`, not `AND`.

Copy in the filter pills suggests it uses `AND`:
![image](https://github.com/user-attachments/assets/77c35a05-0990-4b39-9758-3ce00666a882)

It should say something like "contains any of 2 selections".
This applies to `in` and `notIn` as well.

Reply:
Indeed, when you select multiple values in `Contains` or `Is` GUI filters, you get all records that match with at least 1 value (= value1 OR = value2 ...).
When you select multiple values in `Does not contain` or `Is not` GUI filters, you get all records that match none of these values (!= value1 AND != value2...).
So yes, the copy is a bit confusing.